src/components/Footer/Footer.js

```diff
@@ -4,14 +4,16 @@ const StyledFooter = styled.footer`
   position: fixed;
   bottom: 0;
   width: 100%;
-  padding: 25px;
+  padding: 20px 20px 10px 20px;
   background-color: #2ec4b6;
+  color: #063539;
+  text-align: right;
 `;
 
-export default function Footer({ footer }) {
+export default function Footer() {
   return (
     <>
```

Comment:
We don't need the `fragment` in this case as we are only returning the `StyledFooter`